php/commands/site.php

```diff
@@ -830,8 +830,12 @@ public function _list( $args, $assoc_args ) {
 		}
 		$sites = get_all_sites( $where );
 
-		foreach ( $sites as $site ) {
-			EE::info( $site['sitename'] );
+		if (empty($sites)){
```

Comment:
@bnap00 Can you please test this ? as per this we will never get site list. I think you want to show error in `if ( empty( $sites ) )` condition and in else you'll list the site-name.